src/main/java/ca/team2706/frc/robot/config/Config.java

```diff
@@ -65,21 +65,77 @@ public static void init() {
             INVERT_BACK_RIGHT_DRIVE = robotSpecific(true, true, true);
 
-    public static final boolean DRIVEBASE_CURRENT_LIMIT = robotSpecific(false, false, false);
+    public static final int
+            INTAKE_MOTOR_ID = robotSpecific(6, 6, 6),
+            CARGO_IR_SENSOR_ID = robotSpecific(3, 3, 3),
+            INTAKE_LIFT_SOLENOID_FORWARD_ID = robotSpecific(2, 2, 2),
+            INTAKE_LIFT_SOLENOID_BACKWARD_ID = robotSpecific(3, 3, 3),
+            HATCH_EJECTOR_SOLENOID_FORWARD_ID = robotSpecific(0, 0, 0),
+            HATCH_EJECTOR_SOLENOID_BACKWARD_ID = robotSpecific(1, 1, 1);
+
+    public static final int
+            LIFT_MOTOR_ID = robotSpecific(5, 5, 5);
+
+    public static final double
+            // Max speed of the lift going up in override (between 0 and 1).
+            LIFT_OVERRIDE_UP_SPEED = 0.4,
+    /**
+     * Speed for automatically ejecting cargo from the intake, from 0 to 1.
+     */
+    AUTO_EJECT_CARGO_INTAKE_SPEED = 1.0,
+    // Max speed of the lift going down in override (between -1 and 0).
+    LIFT_OVERRIDE_DOWN_SPEED = -0.2,
+    /**
+     * Speed (from 0 to 1) for automatically intaking cargo.
+     */
+    AUTO_INTAKE_CARGO_SPEED = 0.8;
+
+
+    public static boolean
+            INVERT_LIFT_MOTOR = robotSpecific(false, false, false),
+            ENABLE_LIFT_CURRENT_LIMIT = robotSpecific(false, true, true);
+
+    public static int
+            /**
+             * Maximum lift current, in amps.
+             */
+            MAX_LIFT_CURRENT = 16,
+    /**
+     * How long the lift current has to be over the current limit before it is cut out.
+     */
+    CURRENT_LIMIT_THRESHOLD_MS = 1000,
+    /**
+     * Continuous current limit
+     */
+    CONTINUOUS_CURRENT_LIMIT = 4;
+
+    public static int MAX_LIFT_ENCODER_TICKS = 54_000;
+
+
+    public static final boolean ENABLE_DRIVEBASE_CURRENT_LIMIT = robotSpecific(false, false, false);
 
     // Talon ID for the Pigeon
     public static final int GYRO_TALON_ID = robotSpecific(5, 3, 5);
 
     // Selector Channel
     public static final int SELECTOR_ID = robotSpecific(0, 0, 0);
 
+    // Ring light
+    public static final int RING_LIGHT_ID = robotSpecific(0, 0, 0);
+
     // The amount of encoder ticks that the robot must drive to go one foot
     public static final double DRIVE_ENCODER_DPP
             = robotSpecific(Math.PI / 8192.0, Math.PI / 8192.0, Math.PI / 8192.0);
 
+    /**
+     * The amount of encoder ticks that the robot must move the lift to travel one foot
+     */
+    public static final double LIFT_ENCODER_DPP
+            = robotSpecific(0.7 * Math.PI / 16_384.0, 0.7 * Math.PI / 16_384.0, 0.7 * Math.PI / 16_384.0);
+
     public static final double PIGEON_DPP = robotSpecific(360.0 / 8192.0, 360.0 / 8192.0, 360.0 / 8192.0);
 
-    public static final boolean ENABLE_CAMERA = robotSpecific(true, true, false);
+    public static final boolean ENABLE_CAMERA = robotSpecific(true, true, true);
```

Comment:
We have this disabled in case the simulation robot because there won't be a camera. Is there a reason that you changed it?

Reply:
I thought robot id 2 was for comp bot?